eclass/kernel-install.eclass

```diff
@@ -757,8 +757,10 @@ kernel-install_compress_modules() {
 	if use modules-compress; then
 		einfo "Compressing kernel modules ..."
 		# taken from scripts/Makefile.modinst
-		find "${ED}/lib" -name '*.ko' -exec \
-			xz --check=crc32 --lzma2=dict=1MiB {} + || die
+		find "${ED}/lib" -name '*.ko' -print0 |
+			xargs -0 -P "$(makeopts_jobs)" -n 128 \
+				xz --check=crc32 --lzma2=dict=1MiB
```

Comment:
`linux-mod-r1.eclass` does:

```
	compress=(
			xz -q
			--memlimit-compress=50%
			--threads="$(makeopts_jobs)"
			# match options from kernel's Makefile.modinst (bug #920837)
			--check=crc32
			--lzma2=dict=1MiB
		)
```

Could we do the same here (i.e. use `--threads` and maybe `--memlimit-compress` as well).

Reply:
`--threads` is useless, as it controls compression for a single file. We have lots of small files, not one huge file.

Reply:
> `--threads` is useless, as it controls compression for a single file. We have lots of small files, not one huge file.

Right, that makes sense. LGTM then.

Reply:
Now now, it's useful when it compresses the 61MB nvidia.ko :eyes:

Reply:
I know. I was referring to the specific case of kernel ebuild.